docs/components/tooltip.md

```diff
@@ -30,8 +28,28 @@ Here you can pass in the text to display in the toolip.
 </KTooltip>
 ```
 
+When using the `label` prop (or the [`content` slot](#content)), passing a value of `undefined`, an empty string, or no `content` slot content will prevent the tooltip from showing, while still displaying your `default` slot content.
+
+<KTooltip :label="labelProptooltipText">
+  <KButton>Hover me</KButton>
+</KTooltip>
+
+```html
+<KTooltip :label="tooltipLabel">
+  <KButton>Hover me</KButton>
+</KTooltip>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+
+// The tooltip doesn't have label text yet,
+// so hovering over the button will not render an empty tooltip
+const tooltipLabel = ref<string>('')
+</script>
+```
+
 :::warning NOTE
-KTooltip won't be triggered if the element you pass through `default` slot has `disabled` attribute. You can get around that by wrapping your disabled element around an additional tag, like shown in the example below. 
+KTooltip won't be triggered if the element you pass through `default` slot has `disabled` attribute. You can get around that by wrapping your disabled element around an additional tag, like shown in the example below.
```

Comment:
Unrelated to your changes but is this still true??

![image](https://github.com/Kong/kongponents/assets/67973710/cf5d13ab-25a4-4f7b-8085-b28044a666bf)

Reply:
Chatted with Max about this as well -- removed